core/src/main/java/org/jdbi/v3/core/collector/OptionalBuilder.java

```diff
@@ -20,21 +20,21 @@
     private final Supplier<O> empty;
     private final Function<T, O> factory;
 
-    boolean set;
-    T value;
+    private boolean set;
```

Comment:
technically not backward-compatible, but this isn't really API, so what do we do?

Reply:
Demoting from package-private to private is fine. You just can't demote from API to non-API

Reply:
slipped my mind that the `jdbi` package tree is personal space :)